apps/hash-agents/app/logger/http.py

```diff
@@ -41,9 +39,13 @@ async def logging_middleware(request: Request, call_next: Callable) -> Response:
 
         # Recreate the Uvicorn access log format,
         # but add all parameters as structured information
-        client = f"{client_host}:{client_port}"
         logger.info(
-            f"""{client} - "{http_method} {url} HTTP/{http_version}" {status_code}""",
+            """{%s:%s} - "{%s} {%s} HTTP/{%s}" {%s}""",
```

Comment:
Specific reason why? `f` strings are faster and AFAIK this is not really something you do w/ structlog


> Like in logging, positional arguments are [interpolated into the message string using %](https://docs.python.org/3/library/stdtypes.html#old-string-formatting). That might look dated, but it’s much faster than using [str.format](https://docs.python.org/3/library/stdtypes.html#str.format) and allows structlog to be used as drop-in replacement for [logging](https://docs.python.org/3/library/logging.html#module-logging). If you know that the log entry is always gonna be logged out, just use [f-strings](https://docs.python.org/3/tutorial/inputoutput.html#formatted-string-literals) which are the fastest.

Reply:
As it says, _If you know that the log entry is *always gonna be logged out*_. We do not, it's dependent on log level. Using f-strings in a logger is a codesmell (and a lint failure).

I agree it's not something you do with structlog, you generally want to actually structure them (e.g. `x=foo`) but this code seems to be trying to match a very specific format